constellations/orion/src/router/handler.go

```diff
@@ -90,5 +90,15 @@ func (h *Handler) SetupApiEndpoints() {
 		apiUsers.DELETE("/user/:id", controllers.DeleteUser)
 		apiUsers.GET("/guardian/:guardianId", controllers.GetUserByGuardian)
 	}
+
+	apiFamilies := h.Engine.Group("api/families")
+	{
+		api.Families.GET("/all", controllers.GetAllFamilies)
+		api.Families.POST("/create", controllers.CreateFamily)
+		api.Families.GET("/family/:id", controllers.GetFamilyById)
+		api.Families.POST("/family/:id", controllers.UpdateFamily)
+		api.Families.DELETE("/family/:id", controllers.DeleteFamily)
+		api.Families.GET("/family/:primaryEmail",controllers.GetFamilyByPrimaryEmail)
```

Comment:
oh hmm I think this will cause some problems later. I don't think you can have a url that looks like:
```
api/families/family/aaron@gmail.com
```
Instead, change the url to:
```
api/families/search
```
And have primaryEmail come in as part of the bodyJson in the controller part (like how create/update do it)